neurolib/models/multimodel/builder/fitzhugh_nagumo.py

```diff
@@ -0,0 +1,179 @@
+"""
+FitzHugh–Nagumo model.
+
+Main references:
+    FitzHugh, R. (1955). Mathematical models of threshold phenomena in the
+    nerve membrane. The bulletin of mathematical biophysics, 17(4), 257-278.
+
+    Nagumo, J., Arimoto, S., & Yoshizawa, S. (1962). An active pulse
+    transmission line simulating nerve axon. Proceedings of the IRE, 50(10),
+    2061-2070.
+
+Additional reference:
+    Kostova, T., Ravindran, R., & Schonbek, M. (2004). FitzHugh–Nagumo
+    revisited: Types of bifurcations, periodical forcing and stability regions
+    by a Lyapunov functional. International journal of bifurcation and chaos,
+    14(03), 913-925.
+"""
+
+import numpy as np
+from jitcdde import input as system_input
+
+from ..builder.base.network import Network, Node
+from ..builder.base.neural_mass import NeuralMass
+
+DEFAULT_PARAMS = {
+    "alpha": 3.0,
+    "beta": 4.0,
+    "gamma": -1.5,
+    "delta": 0.0,
+    "epsilon": 0.5,
+    "tau": 20.0,
+    "ext_input_x": 1.0,
+    "ext_input_y": 0.0,
+}
+
+
+class FitzHughNagumoMass(NeuralMass):
+    """
+    FitzHugh-Nagumo neural mass.
+    """
+
+    name = "FitzHugh-Nagumo mass"
+    label = "FHNmass"
+
+    num_state_variables = 2
+    num_noise_variables = 2
+    coupling_variables = {0: "x", 1: "y"}
+    state_variable_names = ["x", "y"]
+    required_params = [
+        "alpha",
+        "beta",
+        "gamma",
+        "delta",
+        "epsilon",
+        "tau",
+        "ext_input_x",
+        "ext_input_y",
+    ]
+    required_couplings = ["network_x", "network_y"]
```

Comment:
The "coupling type" `x_coupling="diffusive", y_coupling="none"` is an argument of `FitzHughNagumoNetwork`. 

Couldn't it be defined here and stay fixed for the model? It could be easier to automate the coupling then in the superclass (see my comment at end of file in `_synch()`).